components/authz-service/server/v2/policy_refresher.go

```diff
@@ -254,6 +254,8 @@ func (refresher *policyRefresher) getIAMVersion(ctx context.Context) (api.Versio
 		return vsn, err
 	}
 	switch ms {
+	// this case shouldn't happen
```

Comment:
I think it is a bit clearer to say
```suggestion
	// this case should no longer happen
```